Packs/SymantecEndpointProtection/Integrations/SymantecEndpointProtection_V2/SymantecEndpointProtection_V2.py

```diff
@@ -836,5 +836,5 @@ def move_client_to_group_command(token):
         move_client_to_group_command(token)
     if current_command == 'sep-identify-old-clients':
         old_clients_command(token)
-except Exception, ex:
+except Exception as ex:
```

Comment:
This is python 2 integration, not python 3

Reply:
This syntax works in BOTH.